homeassistant/components/soma/const.py

```diff
@@ -4,3 +4,8 @@
 HOST = "host"
 PORT = "port"
 API = "api"
+
+MSG_API_UNREACHABLE = "Connection to SOMA Connect failed"
+MSG_API_REACHABLE = "Connection to SOMA Connect succeeded"
+MSG_DEVICE_UNREACHABLE = "Unable to reach device %s (%s)"
+MSG_DEVICE_REACHABLE = "Device %s is now reachable"
```

Comment:
Usually we don't make logger messages constants since it adds an additional step to find the message.  If you find yourself logging the same message over and over, the usual advice is to move the logging into a function.

Reply:
Fixed
I have removed the constants and used different error messages for each action